src/components/Pages/Login/LoginForm/index.js

```diff
@@ -104,9 +151,9 @@ export const LoginForm = () => {
             </div>
           </div>
 
-          {error && <p className={style.error__text}>{error}</p>}
+          {/* {error && <p className={style.error__text}>{error}</p>} */}
 
-          {!email.length && !password.length ? (
+          {!email.length || !password.length ? (
```

Comment:
This check could be done inside a prop called `disabled` - if true, button would be disabled, if false, button should render with default state